README.md

```diff
@@ -126,6 +127,18 @@ for read in R1 R2 I1 I2; do cat L*/${read}.fastq.gz > ./${read}.fastq.gz; done
 
 FASTQ files _must_ be BGZF compressed.
 
+All reads containing sample barcodes/indexes _must_ be of the same length.
+
+###### Auto-detecting FASTQS from a Path Prefix
+
+Alternatively, the FASTQS can be auto-detected when a path prefix is given to `--fastqs <dir>/<prefix>`.
+The FASTQs must be named `<dir>/<prefix>_L00<lane>_<kind><kind-number>_001.fastq.gz`, where `kind` is
+one of R (read/template), I (index/sample barcode), or U (umi/molecular barcode). 
+
+If the read structure is given on the command line or Sample sheet, the segments are assumed to to
+apply to I1 R1 R2 I2 reads in that order.  Otherwise, the read structure will be `B+ T+ T+ B+` 
```

Comment:
```suggestion
apply to I1 R1 R2 I2 reads in that order.  Otherwise, the read structure will be `+B +T +T +B` 
```

This feels very strange/dangerous to me.  Firstly I would have totally expected a different ordering (R1 I1 I2 R2), but moreso that if we're auto-detecting the FASTQs I don't think we should let the users specify the read-structures - it's just too hard to ensure they are in sync.  Either specify both, or auto both would be my preference.